contract/src/contract.rs

```diff
@@ -48,19 +43,14 @@ pub fn instantiate(
     PAUSED.save(deps.storage, &false)?;
 
     let init_staking_config = msg.staking_config.unwrap_or(StakingConfig {
-        minimum_stake_to_register:               INITIAL_MINIMUM_STAKE_TO_REGISTER,
-        minimum_stake_for_committee_eligibility: INITIAL_MINIMUM_STAKE_FOR_COMMITTEE_ELIGIBILITY,
-        allowlist_enabled:                       false,
+        minimum_stake:     INITIAL_MINIMUM_STAKE,
+        allowlist_enabled: false,
```

Comment:
Should we maybe consider to set this to `true` by default?
It may save us some extra step in the future.

Reply:
We could, do we want to? We are intending to leave default minimum stake as 1 right?

Reply:
Because it is an optional field while instantiating, it does not matter much.